### PR TITLE
Let bumpversion respect the update_history data.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Fixes issue #310.
See also https://github.com/collective/zestreleaser.towncrier/pull/11

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@ Changelog for zest.releaser
 6.16.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Let bumpversion respect the ``update_history`` data.
```
Comment (reinout) [inline CHANGES.rst]: I only understood this one after reading the pull request. I don't use the "update_history" setting myself, that is probably the cause :-)

Suggestion:

```suggestion
- Let bumpversion respect the ``update_history`` setting.
```

Actually, just so I understand it right: that `update_history` is an *existing* setting that is now used more broadly?

Comment (mauritsvanrees) [reply CHANGES.rst]: Correct. We introduced this last year so `zestreleaser.towncrier` can tell prerelease and postrelease (and now bumpversion) to not touch the history file.

Maybe as changelog message:

"Let bumpversion not touch the history file when an extension has set ``update_history`` to false."

Would that be clear enough?

Comment (reinout) [reply CHANGES.rst]: Perhaps be more explicit?

"Better support for zestreleaser.towncrier (and similar extensions): the `update_history` setting is now also respected by the "bumpversion" command.

Comment (mauritsvanrees) [reply CHANGES.rst]: I have adapted the changelog entry with your suggestion.

**Pull request merged**